src/app/src/pages/eval/components/Eval.tsx

```diff
@@ -13,7 +13,7 @@ import { useResultsViewSettingsStore, useStore } from './store';
 import './Eval.css';
 
 interface EvalOptions {
-  fetchId?: string;
+  fetchId?: string | null;
```

Comment:
This should be `fetchId: string | null` because the value will always be defined.

Reply:
that makes sense 👍🏿  and done :)